### PR TITLE
Use _BitScanForward and _BitScanReverse on MSVC

### PR DESCRIPTION
**Pull request by KindDragon**

### CODE REVIEW

File: src/hb-private.hh [hunk 1]
```diff
@@ -352,6 +355,10 @@ _hb_bit_storage (unsigned int number)
 {
 #if defined(__GNUC__) && (__GNUC__ >= 4) && defined(__OPTIMIZE__)
   return likely (number) ? (sizeof (unsigned int) * 8 - __builtin_clz (number)) : 0;
+#elif defined_MSC_VER) && defined(__OPTIMIZE__)
```
Comment (behdad) [inline src/hb-private.hh]: `defined_MSC_VER` missing open parenthesis!

File: src/hb-private.hh [hunk 2]
```diff
@@ -352,6 +355,10 @@ _hb_bit_storage (unsigned int number)
...
+  unsigned long where;
+  if (_BitScanReverse(&where, number)) return 1 + where;
```
Comment (behdad) [inline src/hb-private.hh]: Space before "(" please.

**Comment by ebraminio** [comment]
Fixed, @behdad, please be more careful about not breaking the bots :)

File: src/hb-private.hh (hunk 1 above)
Comment (KindDragon) [reply src/hb-private.hh]: Sorry :)

**Comment by behdad** [comment]
> Fixed, @behdad, please be more careful about not breaking the bots :)

Ugh. I accidentally merged this. I didn't mean to. Meant to comment on it. Ouch. Sorry.

**Pull request merged**